server/errorHandler.ts

```diff
@@ -20,9 +21,33 @@ export default function createErrorHandler(production: boolean) {
       if (err.status === 403 && (!err.external || err.response?.body?.accessErrors)) {
         res.status(403)
 
+        const returnedError = err.response?.body?.message
+        let userMessage
+        let userHeader
+
+        const userHeaderTypes = {
+          userHeaderService: 'You do not have permission to view this service',
+          userHeaderPage: 'You do not have permission to view this page',
+        }
+
+        const lookupError = errorMessages.returnedError[returnedError]
+        if (lookupError) {
+          userMessage = lookupError.mappedMessage
+          userHeader = userHeaderTypes[lookupError.userHeaderType]
+        } else if (returnedError?.includes('unidentified provider')) {
+          userMessage = errorMessages.errorHandlerAccessErrorMessages.providerGroupNotRecognised
+          userHeader = userHeaderTypes.userHeaderService
+        } else if (returnedError?.includes('unidentified contract')) {
+          userMessage = errorMessages.errorHandlerAccessErrorMessages.contractGroupNotRecognised
+          userHeader = userHeaderTypes.userHeaderService
+        } else if (returnedError?.includes('is not accessible to providers')) {
+          userMessage = errorMessages.errorHandlerAccessErrorMessages.groupsDoNotMatch
+          userHeader = userHeaderTypes.userHeaderService
+        }
```

Comment:
still needs the 
```
        } else {
          userHeader = userHeaderTypes.default
        }
```
otherwise it will return userHeader undefined for any unknown error message

Reply:
This works without the else, by default we display the message if it's undefined in authError